ts/components/conversation/UpdateGroupMembersDialog.tsx

```diff
@@ -135,6 +134,19 @@ export class UpdateGroupMembersDialog extends React.Component<Props, State> {
     );
   }
 
+  private renderMemberList() {
+    const members = this.state.friendList;
+
+    return members.map((member: ContactType) => (
+      <SessionMemberListItem
```

Comment:
Should we put in a `key` prop so react renders the list correctly?

Reply:
done